src/main/java/edu/umich/its/cpm/MigrationController.java

```diff
@@ -1729,10 +1740,14 @@ public Response getSiteMigrationInBulkUpload(
 			// now that we get the site ids for batch upload
 			// start the batch process
 			String bulkMigrationId = java.util.UUID.randomUUID().toString();
-
+			
 			for (String siteId : bulkUploadSiteIds) {
 				// for each site id, start the migration process
 				// associate it with the bulk id
+				
+				// add admin user to site with Owner role
```

Comment:
A catch of Exception is added, and in case of any exception, the removeAddedAdminOwner() is called to remove the just-added admin user